xbmc/filesystem/PluginDirectory.h

```diff
@@ -34,6 +34,13 @@ class CPluginDirectory : public IDirectory, public CRunningScriptsHandler<CPlugi
   float GetProgress() const override;
   void CancelDirectory() override;
   static bool RunScriptWithParams(const std::string& strPath, bool resume);
+
+  /*! \brief Get a reproducible CFileItem by trying to resolve the plugin paths up to 5 times.
```

Comment:
Please remove the reference to 5 and use max allowed nested resolutions or similar

Reply:
done